integrations.json

```diff
@@ -612,6 +612,176 @@
       }
     ]
   },
+  {
+    "name": "Kafka",
+    "method_replacements": [
+      {
+        "caller": {},
+        "target": {
+          "assembly": "Confluent.Kafka",
+          "type": "Confluent.Kafka.Consumer`2",
+          "method": "Close",
+          "signature_types": [
+            "System.Void"
+          ],
+          "minimum_major": 1,
+          "minimum_minor": 4,
+          "minimum_patch": 0,
+          "maximum_major": 1,
+          "maximum_minor": 65535,
+          "maximum_patch": 65535
+        },
+        "wrapper": {
+          "assembly": "Datadog.Trace.ClrProfiler.Managed, Version=1.26.2.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
```

Comment:
1.26.3 is already released. Can you merge/rebase `master` and re-generate this file?

Reply:
If we ran the integration tests on .NET Framework, they would fail since the assembly with exact version 1.26.2 would not be found.

Reply:
Can't keep up with the pace of releases atm 😂

Reply:
updated in https://github.com/DataDog/dd-trace-dotnet/pull/1444/commits/57a4a98b60245400ef1254a75d45c632e732611b